src/debugging/README.md

```diff
@@ -1,4 +1,11 @@
-To run in Google Chrome:
+# Attach on Run from Xcode
+1. `cd src/debugging`
+1. `./launch-proxy.sh`
+1. In Xcode add `--nativescript-debug-brk` command line arg in the app scheme of the NativeScript application to be debugged
+1. Open WebInspectorUI/Main.html in latest Safari version
```

Comment:
The latest Safari version may not be the right one. When you update JavaScriptCore you should also update the debugging frontend which may require features that are included only in the nightly Safari version that uses that specific JavaScriptCore and are missing from the official one :)

Reply:
Thanks for the clarification, @KristinaKoeva :) I've added the following paragraph below. Does it sound better this way?

###### Note that if you've just updated the WebKit version even the canary version of Safari may be missing some features used by the frontend. The latest nightly Safari version that uses that specific JavaScriptCore must be used in this case. Updating and re-building `src/debugging/Inspector/Inspector` application should always work as well.